src/sdk/accountspace_fs.js

```diff
@@ -26,6 +27,26 @@ const { account_cache } = require('./object_sdk');
 const dummy_region = 'us-west-2';
 const dummy_service_name = 's3';
 
+const account_id_cache = new LRUCache({
+    name: 'AccountIDCache',
+    // TODO: Decide on a time that we want to invalidate
+    expiry_ms: Number(config.ACCOUNTS_ID_CACHE_EXPIRY),
+    /**
+     * @param {{ _id: string, config_fs: import('./config_fs').ConfigFS, silent_if_missing?: boolean }} params
+     */
```

Comment:
I don't see the param `silent_if_missing` used.
I think it will be easier to read if every param will be in a new line, for example:
```suggestion
    /**
     * @param {{ 
                   _id: string,
                   config_fs: import('./config_fs').ConfigFS,
         }} params
     */
```

Reply:
thanks I removed it, it's a leftover of code I removed